NGitLab.Mock/Config/GitLabPermission.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NGitLab.Models;
+
+namespace NGitLab.Mock.Config
+{
+    /// <summary>
+    /// Describe a permission in a GitLab group/project
+    /// </summary>
+    public class GitLabPermission : GitLabObject
+    {
+        /// <summary>
+        /// Username (required if user permission)
+        /// </summary>
+        public string User { get; set; }
+
+        /// <summary>
+        /// Group fullname (required is group permission)
+        /// </summary>
+        public string Group { get; set; }
+
+        public AccessLevel Level { get; set; } = AccessLevel.Developer;
```

Comment:
The value is already set by the extension methods

```suggestion
        public AccessLevel Level { get; set; }
```